Sources/Stats/py_stats.cc

```diff
@@ -127,6 +129,77 @@ void AddStatsModule(py::module m) {
                     Rank-2 tensors should have shape `(N, M)` where `N` is the
                     number of samples in one Markov Chain and `M` is the number
                     of Markov Chains. Data should be in row major order.)EOF");
+
+  subm.def(
+      "covariance_sv",
```

Comment:
I guess this function would be fast enough if written in pure python with numpy?

Reply:
Since it performs an MPI reduction, I think it's better keep the C++ at the moment.